neurodamus/commands.py

```diff
@@ -55,6 +55,7 @@ def neurodamus(args=None):
         --enable-shm=[ON, OFF]  Enables the use of /dev/shm for coreneuron_input [default: ON]
         --model-stats           Show model stats in CoreNEURON simulations [default: False]
         --dry-run               Dry-run simulation to estimate memory usage [default: False]
+        --prosp-hosts=<number>  Number of prospective hosts for dry-run load balancing [default: 40]
```

Comment:
Can we have a different name for this option? `prosp` seem like a very weird abbreviation. Also, `hosts` would make me think of machine _names_. 

Looking at the rest of the code, this seems to be more the _number of targeted ranks_?

Reply:
Yeah, I never liked the name. I used that since it allowed a quick "hijack" of an older unused CLI variable. Will rename to something more akin to what you suggested, thanks!